polymer.externs.js

```diff
@@ -41,6 +41,24 @@ PolymerElement.prototype.isAttached;
 PolymerElement.prototype.root;
 
 /**
+ * The shadow root for the element. 
+ * Only exists if running under Shadow Dom.
+ * You usually want to use `this.root`.
+ *
+ * @type {Node|undefined}
+ */
+PolymerElement.prototype.shadowRoot;
```

Comment:
Turns out that shadowRoot actually lives on HTMLElement, so this extern isn't necessary: https://github.com/google/closure-compiler/blob/master/externs/browser/html5.js#L1186